exchanges/btse/btse_test.go

```diff
@@ -37,14 +38,33 @@ func TestSetup(t *testing.T) {
 	b.Setup(&btseConfig)
 }
 
+func TestGetMarketsSummary(t *testing.T) {
+	b.SetDefaults()
+	_, err := b.GetMarketsSummary()
+	if err != nil {
+		t.Fatalf("Test failed. Err: %s", err)
+	}
+}
+
 func TestGetMarkets(t *testing.T) {
 	b.SetDefaults()
+	b.Verbose = true
```

Comment:
I didn't bother mentioning these as I assume you're still using these for testing in your PR, but before merging these + the t.Logs need to be removed from all funcs